tests/storage/hotplug.go

```diff
@@ -1156,17 +1166,23 @@ var _ = SIGDescribe("Hotplug", func() {
 	Context("virtctl", func() {
 		var (
 			vm *v1.VirtualMachine
+			sc string
 		)
 
 		BeforeEach(func() {
+			var exists bool
+			sc, exists = tests.GetStorageClassWithBindingModeWaitForFirstConsumer()
+			if !exists {
+				Skip("Skip no wffc storage class available")
+			}
 			vm = createAndStartWFFCStorageHotplugVM()
 		})
 
 		table.DescribeTable("should add volume according to options", func(dryRun bool) {
 			vmi, err := virtClient.VirtualMachineInstance(vm.Namespace).Get(vm.Name, &metav1.GetOptions{})
 			Expect(err).ToNot(HaveOccurred())
 			tests.WaitForSuccessfulVMIStartWithTimeout(vmi, 240)
-			dv := tests.NewRandomBlankDataVolume(util.NamespaceTestDefault, tests.Config.StorageClassLocal, "64Mi", corev1.ReadWriteOnce, corev1.PersistentVolumeFilesystem)
+			dv := tests.NewRandomBlankDataVolume(util.NamespaceTestDefault, sc, "64Mi", corev1.ReadWriteOnce, corev1.PersistentVolumeFilesystem)
```

Comment:
The storage class can be the tests.Config.RWOFilesystem

Reply:
I will use the storage class from the before each but I changed the function to check for tests.Config.RWOFilesystem and check it is wffc